<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZeroLevelSetting</class>
 <widget class="QWidget" name="ZeroLevelSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(27, 150, 208);
font-size:24px;
}
QGroupBox{
border:1px solid green;
font-size:20px;
color:white;
}
QPushButton{
border-radius:10;
background-color: rgb(80, 80, 80);
font-size:20px;
color:rgb(255, 255, 0);
}
QPushButton::pressed{
background-color: rgb(120, 120, 120);
}
QPushButton::checked{
color:red;
}</string>
  </property>
  <widget class="QPushButton" name="auto_Zero_pb">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>100</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>自
动</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Zero_pb">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>385</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>返
回</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-style:outset;
background: transparent;
font:bold;
font-size:22px;
color:blue;
}
QPushButton::checked{
color:red;
}</string>
   </property>
   <property name="title">
    <string>实际脉冲</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;总脉冲值&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;总针数值&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;当前脉冲&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;当前针数&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pulse_lb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="needle_lb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="currentpulse_lb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="currentneedle_lb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;零位脉冲&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeropluse_pb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size:22px;color:red;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="save_pb">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>290</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>保
存</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-style:outset;
background: transparent;
font:bold;
font-size:22px;
color:blue;
}
QPushButton::pressed{
color:red;
}
QPushButton::checked{
color:red;
}</string>
   </property>
   <property name="title">
    <string>设定脉冲</string>
   </property>
   <widget class="QPushButton" name="sumPluseSetting_btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>40000</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroNeedleSetting_btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;总脉冲值&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;零位针值&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushbutton_set">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>195</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>设
定</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="auto_Zero_pb_2">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>5</y>
     <width>55</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>手
动</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fixZeroOffset">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-style:outset;
background: transparent;
font:bold;
font-size:22px;
color:blue;
}
QPushButton::checked{
color:red;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>机械零位偏差</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
